### ISSUE TITLE
Missing SVG contentType in ContentTypeResolver

### ISSUE THREAD

**Issue by M3lkior** [alexo/wro4j#216]
defaultContentTypeMap in ContentTypeResolver doesn't contain svg type.

Could you add it in next release ?

here is the correct content type to put in map : 

<code>		defaultContentTypeMap.put("svg", "image/svg+xml");
</code>

thanks :)

**Comment by alexo** [comment]
Thanks for reporting this issue.

**Comment by alexo** [comment]
Fixed in branch 1.7.x

**Comment by M3lkior** [comment]
You can also add :

```
// font-type
		defaultContentTypeMap.put("ttf", "application/x-font-ttf");
		defaultContentTypeMap.put("woff", "application/font-woff");
		defaultContentTypeMap.put("woff2", "application/font-woff2");
```

**Comment by alexo** [comment]
I'll update the implementation. Reopening the issue. 
Thanks for pointing this out.

**Comment by alexo** [comment]
Btw, the project is moving to a new location:github.com/wro4j/wro4j
This is where the issues from google code are already migrated. The documentation will be migrated later as well....

**Comment by M3lkior** [comment]
I had not seen. you should highlight this moving :)
Too bad, wro4j will lose its stars :/

**Comment by alexo** [comment]
Indeed, there is no way to preserve them. I've decided to move the project, since google code is closing and all the issues were located there. Also consolidating issues, documentation and code in a single location is a good idea on the long run..



—
